README.md

```diff
@@ -1,20 +1,25 @@
-YAGO 4 pipeline
-===============
+YAGO 4 (Pipeline)
+=================
 
-[![actions status](https://github.com/yago-naga/yago4/workflows/build/badge.svg)](https://github.com/yago-naga/yago4/actions)
+[![actions status][1]][2] [![crates.io version][3]][4] [![docs.rs][5]][6] [![dependency status][7]][8] ![GNU GPLv3][gpl-logo]
```

Comment:
The GPL badge is huge compared to the other

Reply:
https://www.gnu.org/graphics/license-logos.html

Took the smallest one possible =/

Reply:
What about just ![GPLv3+](https://img.shields.io/github/license/yago-naga/yago4) ?

```
![GPLv3+](https://img.shields.io/github/license/yago-naga/yago4)
```

Reply:
/done